vm/array.go

```diff
@@ -77,7 +77,7 @@ func initializeArray(elements []Object) *ArrayObject {
 	return &ArrayObject{Elements: elements, Class: arrayClass}
 }
 
-func init() {
+func initArray() {
```

Comment:
The init method is Go's built in method that will be executed when the package is imported.